Latest release 10.0.0 does not compile

The latest release does not compile out of the box, see below for details:

```shell
❯ cargo init foo
    Creating binary (application) package
note: see more `Cargo.toml` keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

❯ cargo add yup-oauth2
    Updating crates.io index
      Adding yup-oauth2 v10.0.0 to dependencies
             Features:
             + hyper-rustls
             + rustls
             + rustls-pemfile
             + service_account
             - hyper-tls
    Updating crates.io index

❯ cargo run
   Compiling yup-oauth2 v10.0.0
error[E0599]: no method named `with_native_roots` found for struct `HttpsConnectorBuilder` in the current scope
    --> /Users/davidk/.cargo/registry/src/index.crates.io-6f17d22bba15001f/yup-oauth2-10.0.0/src/authenticator.rs:1007:14
     |
1006 |           let connector = hyper_rustls::HttpsConnectorBuilder::new()
     |  _________________________-
1007 | |             .with_native_roots()?
     | |             -^^^^^^^^^^^^^^^^^ method not found in `HttpsConnectorBuilder<WantsTlsConfig>`
     | |_____________|
     |

error[E0599]: no method named `with_native_roots` found for struct `HttpsConnectorBuilder` in the current scope
    --> /Users/davidk/.cargo/registry/src/index.crates.io-6f17d22bba15001f/yup-oauth2-10.0.0/src/authenticator.rs:1027:14
     |
1026 |           let connector = hyper_rustls::HttpsConnectorBuilder::new()
     |  _________________________-
1027 | |             .with_native_roots()
     | |             -^^^^^^^^^^^^^^^^^ method not found in `HttpsConnectorBuilder<WantsTlsConfig>`
     | |_____________|
     |

For more information about this error, try `rustc --explain E0599`.
error: could not compile `yup-oauth2` (lib) due to 2 previous errors

❯ cat Cargo.toml
[package]
name = "foo"
version = "0.1.0"
edition = "2021"

[dependencies]
yup-oauth2 = "10.0.0"
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,7 +39,7 @@ http = "1"
 http-body-util = "0.1"
 hyper = "1"
 hyper-util = { version = "0.1.5", features = ["client-legacy", "server-auto", "http1", "http2", "server-graceful"] }
-hyper-rustls = { version = "0.27", optional = true, default-features = false, features = ["http2"] }
+hyper-rustls = { version = "0.27", optional = true, default-features = false, features = ["http1", "http2", "rustls-native-certs", "ring"] }
 hyper-tls = { version = "0.6.0", optional = true }
 itertools = "0.13"
 log = "0.4"

